test/integrationTests/testAssets/slnWithCsproj/test/test.csproj

```diff
@@ -1,11 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net8.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
nit: Should this be in a props file somewhere just to keep in sync? Now that the synchronization between them actually matters

Reply:
good call, I can do that.